### PR TITLE
Feature/import proptypes properly

### PR DESCRIPTION
**Pull request by DarioG**

### CODE REVIEW

**Review by arijitdasgupta — APPROVED** [review]
Looks Good! :rocket:

**Review by alanbasse — APPROVED** [review]
PR works with SDX so fine for me. 

Just one thing: We strive for alphabetic ordering within the imports as well, sorted by lib name like `'prop-types'` before `'react'`. It's not fully embraced yet anyway but in regards of uncle Bob...

File: src/framework/appHeader/AppHeader.jsx [hunk 1]
```diff
@@ -1,7 +1,6 @@
 
-import React, {
-  PropTypes,
-} from 'react';
+import React from 'react';
+import PropTypes from 'prop-types';
```
Comment (jenslonkowski) [inline src/framework/appHeader/AppHeader.jsx]: You missed this one ;-)

File: src/framework/chart/LineChart.jsx [hunk 2]
```diff
@@ -1,9 +1,9 @@
 
 import d3 from 'd3';
 import $ from 'jquery';
```
Comment (jenslonkowski) [inline src/framework/chart/LineChart.jsx]: Maybe fix the order here as well ;-)

File: src/framework/viewHeader/dateRange/DateRange.jsx [hunk 3]
```diff
@@ -1,12 +1,12 @@
...
 import classNames from 'classnames';
 import moment from 'moment';
+import PropTypes from 'prop-types';
 import React, {
   Component,
-  PropTypes,
 } from 'react';
-import { DateRange as ReactDateRange } from 'react-date-range';
 
+import { DateRange as ReactDateRange } from 'react-date-range';
```
Comment (jenslonkowski) [inline src/framework/viewHeader/dateRange/DateRange.jsx]: Why did you move this line down?

File: src/guide/components/sourceCodeViewer/SourceCodeViewer.jsx [hunk 4]
```diff
@@ -1,11 +1,11 @@
 
-import React, {
-  Component,
-  PropTypes,
-} from 'react';
 import $ from 'jquery';
```
Comment (jenslonkowski) [inline src/guide/components/sourceCodeViewer/SourceCodeViewer.jsx]: "jquery" needs to go below "highlight.js"

File: src/guide/views/AppView.jsx [hunk 5]
```diff
@@ -3,13 +3,13 @@
 // NOTE: `import` allows us to tersely specify the individual members we want
 // to import from a module with the curly brace syntax and by delimiting
 // them with commas.
+import $ from 'jquery';
```
Comment (jenslonkowski) [inline src/guide/views/AppView.jsx]: "jquery" needs to go below "classnames"

**Review by jenslonkowski — CHANGES_REQUESTED** [review]
Just some minor order fixes required. Good job Dario!

File: src/framework/base/dropdown/BaseDropdown.jsx [hunk 6]
```diff
@@ -13,11 +13,11 @@
  */
 
 import $ from 'jquery';
```
Comment (jenslonkowski) [inline src/framework/base/dropdown/BaseDropdown.jsx]: "jquery" needs to go below "classnames"

File: src/guide/components/sourceCodeViewer/SourceCodeViewer.jsx [hunk 7]
```diff
@@ -1,11 +1,11 @@
 
+import classNames from 'classnames';
+import $ from 'jquery';
```
Comment (jenslonkowski) [inline src/guide/components/sourceCodeViewer/SourceCodeViewer.jsx]: ...hij... -> one more line down please ;-)

**Review by jenslonkowski — APPROVED** [review]
Great work!

**Pull request merged**